digdag-standards/src/main/java/io/digdag/standards/command/CommandExecutorModule.java

```diff
@@ -4,17 +4,19 @@
 import com.google.inject.Module;
 import com.google.inject.Scopes;
 import io.digdag.spi.CommandExecutor;
-import io.digdag.standards.command.kubernetes.DefaultKubernetesClientFactory;
-import io.digdag.standards.command.kubernetes.KubernetesClientFactory;
```

Comment:
We remove KubernetesCommandExecutor ?

Reply:
That good point. I don't remove Kubernetes command executor. But I'm now removing Kubernetes' one from the chain of command executor fall back. So, in this PR, we will have the fall back chain: Ecs -> Docker -> Simple.

I think that we'd better to make another mechanism for loading custom command executors and choose some of them like https://github.com/treasure-data/digdag/issues/901 separately.